target/linux/ath79/base-files/etc/board.d/01_leds

```diff
@@ -102,6 +102,10 @@ glinet,gl-ar300m-lite)
 glinet,gl-x750)
 	ucidef_set_led_netdev "wan" "WAN" "$boardname:green:wan" "eth1"
 	;;
+kuwfi,cpe206)
+	ucidef_set_led_netdev "lan" "LAN" "$boardname:green:lan" "eth0"
+	ucidef_set_led_netdev "wlan" "WLAN" "$boardname:green:wlan" "wlan0"
```

Comment:
You could use phy0tpt trigger for wlan led here as well. Just add the trigger line in dts and drop this line.
And by dropping wlan trigger, this device got identical led setup as alfa-network,ap121f. It can be added in this case block at line 15:
```
alfa-network,ap121f)
	ucidef_set_led_netdev "lan" "LAN" "$boardname:green:lan" "eth0"
	;;
```

Reply:
done